cypress/support/commands.js

```diff
@@ -63,8 +66,25 @@ Cypress.Commands.add('createArticle', (title, description, body) => {
         }
       },
       headers: {
-        Authorization: `Token ${authToken}`
+        Authorization: `Token ${authToken}` 
       }
     });
   });
 });
+
+Cypress.Commands.add('deleteArticle', (slug) => {
+  cy.getCookie('auth').then((token) => {
+    if (!token) {
+      throw new Error('Auth cookie not found. Make sure you are logged in.');
+    }
```

Comment:
Same here, consider displaying a user-friendly message instead of throwing an error.